django_celery_beat/models.py

```diff
@@ -138,6 +132,7 @@ class Meta:
         verbose_name = _('interval')
         verbose_name_plural = _('intervals')
         ordering = ['period', 'every']
+        unique_together = ('every', 'period')
```

Comment:
what about using uniqueConstraint instead?

Reply:
[`UniqueConstraint`](https://docs.djangoproject.com/en/3.0/ref/models/constraints/#uniqueconstraint) was added in Django 2.2. while django-celery-beat says to support older versions than that according to https://github.com/celery/django-celery-beat/blob/4fff14fb15574b5265529d56dc31cef403e339cb/setup.py#L51-L55

Also, `UniqueConstraint` seems to be more verbose. Which feature of `UniqueConstraint` were you looking for that `unique_together` does not offer?